src/Writers/LaTeXWriter.jl

```diff
@@ -498,8 +498,15 @@ function _print_code_escapes_inline(io, s::AbstractString)
 end
 
 function latex(io::IO, md::Markdown.Paragraph)
-    for md in md.content
-        latexinline(io, md)
+    if occursin(r"^\h*?\\begin{\p{Xan}*?}", md.content[begin])
```

Comment:
The `[begin]` syntax was added in 1.4, so this would have to be rewritten as e.g. `first(md.content)` to be compatible with 1.0.